recipes/libx265/all/conanfile.py

```diff
@@ -1,4 +1,5 @@
 from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
```

Comment:
```suggestion
```

PR title could be updated (it doesn't disallow anymore assembly with shared or fPIC).

Reply:
Indeed, thanks for the review. Now review again :stuck_out_tongue: